tests/sentry/options/test_manager.py

```diff
@@ -18,7 +18,9 @@
 class OptionsManagerTest(TestCase):
     @fixture
     def store(self):
-        return OptionsStore(cache=RedisCache())
+        return OptionsStore(
+            cache=RedisCache()
+        )
```

Comment:
I'm not sure if there is any reason not to use the `sentry.options.default_store` here (same goes for manager?)

Since the manager is using the `settings` global, and the local cache should be flushed between test runs (note: should verify this) I don't think there should be any differences in the behavior.